src/components/SchemaViewer/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Fragment } from 'react';
+import { shape, string, object } from 'prop-types';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import { ThemeProvider, createMuiTheme } from '@material-ui/core';
+import SchemaTable from '../SchemaTable';
+
+function SchemaViewer({ schema, theme }) {
+  const MuiTheme = createMuiTheme(theme);
+
+  /** TODO: add source viewing option */
+  return (
+    <Fragment>
+      <CssBaseline />
+      <ThemeProvider theme={MuiTheme}>
+        <SchemaTable schema={schema} />
+      </ThemeProvider>
+    </Fragment>
+  );
+}

```

Comment:
From https://material-ui.com/customization/theming/#theme-provider:

> If you wish to customize the theme, you need to use the ThemeProvider component in order to inject a theme into your application. However, this is optional; Material-UI components come with a default theme.

It turns out that material-ui ships with the default theme without having to use the ThemeProvider component. Because of this, we won't need to add `ThemeProvider` and `CssBaseline` in `src/components/<component-name>` since all of our component will use the default theme by default. A user will be able to customize the theme by doing something similar to what we did in `src/index.jsx`, but in their own app:

```jsx
<CssBaseline />
<ThemeProvider theme={theme}>
  <SchemaTable />
</ThemeProvider>
```

Reply:
Ah, now I completely understand how customization works! I initially thought that we had to export a component wrapped by the `<ThemeProvider>` in order to enable users to customize. I understand why we should revert back to the original way things were. 💯 Thanks for clarifying this part!